testdata/project-v2/api/v1/groupversion_info.go

```diff
@@ -15,8 +15,8 @@ limitations under the License.
 */
 
 // Package v1 contains API Schema definitions for the crew v1 API group
-//+kubebuilder:object:generate=true
-//+groupName=crew.testproject.org
+// +kubebuilder:object:generate=true
+// +groupName=crew.testproject.org
```

Comment:
@rumstead could you pelase revert this change? 

Note that you need to run "make generate" for any PR that changes the scaffolds otherwise it will fail in our check done to ensure that sample testdata is updated accordingly : https://github.com/kubernetes-sigs/kubebuilder/runs/8071680207?check_suite_focus=true

Reply:
that change is a result of make generate. 

> > Will do, but why not do that at merge time?
> > So that we have for example a clear and concise git history and if we need to revert the changes it can easily done as well.

Hmm, I don't follow how squashing on merge wouldn't satisfy that requirement. Maybe a convo for a different day. 
https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests

Reply:
Hi @rumstead,

> https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests

It does not bring the same result.

Reply:
```
kubebuilder feat/add-envtest-path                                                                                                                                                                                                                                                                                                                                                                                                                                     
❯ git status
On branch feat/add-envtest-path
Your branch is up to date with 'origin/feat/add-envtest-path'.

nothing to commit, working tree clean

kubebuilder feat/add-envtest-path                                                                                                                                                                                                                                                                                                                                                                                                                                      
❯ make generate
rm -rf testdata/
./test/testdata/generate.sh
Building kubebuilder
...

kubebuilder feat/add-envtest-path                                                                                                                                                                                                                                                                                                                                                                                                                                     
❯ git status
On branch feat/add-envtest-path
Your branch is up to date with 'origin/feat/add-envtest-path'.

nothing to commit, working tree clean

```

Reply:
found it. thanks for the links!

Reply:
seems to be a difference in go versions. I am on go1.19 but when I switch back to go1.18 the spaces are removed.

Reply:
@rumstead we do not support go 1.19 yet. 
That definitely shows to be the case. 
Thank you for checking that.